rpcs3/Ini.h

```diff
@@ -472,11 +443,8 @@ class Inis
 	{
 		switch (code)
 		{
-		case 0: return "WARP";
-		case 1: return "Default";
-		case 2: return "Renderer 0";
-		case 3: return "Renderer 1";
-		case 4: return "Renderer 2";
+		case 0: return "Default";
```

Comment:
The adaptater's name did change a while back, but I forgot to fix it in the log part.